src/components/common/ClientIntakeForm/IntakeForms/Finances.js

```diff
@@ -1,175 +1,189 @@
-                  </Checkbox>
-                </Form.Item>
-                <Form.Item>
-                  <Checkbox value="poorCredit">
-                    <label>No Credit History</label>
-                  </Checkbox>
-                </Form.Item>
-                <Form.Item>
-                  <Checkbox value="noRentalHistory">
-                    <label>No Rental History</label>
-                  </Checkbox>
-                </Form.Item>
-              </Form.Item>
-            </div>
-          </section>
-
-          <section>
-            <div className="Finances_Form_Container_Styles">
-              <Form.Item className="Finances_Form_Container_Styles_Subsection1">
-                <label className="Finances_Form_Label">
-                  Sources of income:
-                </label>
-                <Form.Item>
-                  <Checkbox value="eviction">
-                    <label>TANF</label>
-                  </Checkbox>
-                </Form.Item>
-                <Form.Item>
-                  <Checkbox value="SSI">
-                    <label>SSI</label>
-                  </Checkbox>
-                </Form.Item>
-                <Form.Item>
-                  <Checkbox value="SSI">
-                    <label>SSI</label>
-                  </Checkbox>
-                </Form.Item>
-                <Form.Item>
-                  <Checkbox value="SSDI">
-                    <label>SSDI</label>
-                  </Checkbox>
-                </Form.Item>
-
-                <div className="Finances_Form_Amount_Container">
-                  <Form.Item
-                    label="Amount::"
-                    className="Finances_Form_Amount_Label"
-                  >
-                    <Input
-                      placeholder="$0"
-                      name="Amount"
-                      className="Finances_Form_Amount_Input"
-                    />
+      <Form.Item form={form} className="Finances__Form" layout="inline">
+        <div
+          className="Finances_Form_Sections"
+          style={{
+            display: 'flex',
+            width: 'full',
+            justifyContent: 'space-between',
+          }}
+        >
+          <section style={{ width: '45%' }}>
+            <div className="Finances_Form_Label">Family have history of: </div>
+            <div
+              className="Finances_Form_Container_Styles"
+              style={{ display: 'flex', flexDirection: 'column' }}
+            >
+              {familyHistory.map(info => (
+                <div key={info.name} className="Finances_Form_Container_Styles">
+                  <Form.Item>
+                    <Checkbox
+                      name={info.name}
+                      checked={formValues[info.name]}
+                      value={formValues[info.name]}
+                      onChange={onChange}
+                    >
+                      <label>{info.label}</label>
+                    </Checkbox>
```

Comment:
I like how you cleaned this up and utilized an array to build out the checkboxes instead.  Nice!